modules/administration-guide/pages/configuring-workspace-target-namespace.adoc

```diff
@@ -0,0 +1,62 @@
+:_content-type: PROCEDURE
+:navtitle: Configuring {orch-namespace} name
+:keywords: administration guide, configuring, namespace
+:page-aliases: 
+
+[id="configuring-user-{orch-namespace}-name-for-automatic-provisioning_{context}"]
+= Configuring user {orch-namespace} name for automatic provisioning
+
+You can configure the {orch-namespace} name template that {prod-short} is using to create the required {orch-namespace} when starting a workspace.
+
+A valid {orch-namespace} name template follows this convention:
+
+* The `<username>` or `<userid>` placeholder is mandatory.
+
+* {prod-short} reduces usernames or IDs incompatible with {orch-name} objects naming convention to the nearest valid name by replacing groups of unsuitable symbols with the `-` symbol. 
```

Comment:
```suggestion
* If the formatting of usernames or IDs is incompatible with the naming conventions for {orch-name} objects, {prod-short} changes the usernames and IDs to a valid name by replacing groups of incompatible symbols with the `-` symbol. 
```
- Consider revising the sentence as suggested. The current sentence is awkwardly constructed with some unclear antecedents and is too "noun heavy" at times. 
- Also consider replacing "reduce" with "change" (the change may not be a reduction but merely a replacement) and "unsuitable" with "incompatible", which in this case is more accurate.